src/EnergyPlus/Psychrometrics.hh

```diff
@@ -719,8 +719,8 @@ namespace Psychrometrics {
 		// FUNCTION LOCAL VARIABLE DECLARATIONS:
 
 		Int64 const Tdb_tag( bit::bit_shift( TRANSFER( T, Grid_Shift ), -Grid_Shift ) ); // Note that 2nd arg to TRANSFER is not used: Only type matters
-//		Int64 const hash( bit::bit_and( Tdb_tag, psatcache_mask ) ); //Tuned Replaced by below
```

Comment:
@Myoldmopar. I do not think this change was related to this work.  I do remember going through this section of the code when I was creating unit test for gas humidifier.  I had to include the "Psychomterics" initialization to circumvent the crash of the unit test for gas humidifier.